index.html

```diff
@@ -76,5 +76,7 @@
         <script src="js/vendor/bootstrap.min.js"></script>
         <script src="js/aggregator.js"></script>
         <script src="js/main.js"></script>
+        <script src="http://cdnjs.cloudflare.com/ajax/libs/moment.js/2.9.0/moment.min.js"></script>
+        <script src="http://cdnjs.cloudflare.com/ajax/libs/angular-moment/0.9.0/angular-moment.min.js"></script>
```

Comment:
Can we/do we want to be/is it necessary to be dependent on these two particular versions (i.e. akin to pinning, in Python), or can we safely assume the latest version(s) will be backwards-compatible?

To be clear, there's precedent here in https://github.com/mozilla/mozwebqa-dashboard/blob/18f04e789f0c2542073ad12b72e3660630257a59/index.html#L71-L73 so it's an honest (i.e. not snarky!) question :-)